### ISSUE TITLE
2025-05-14, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#169]
# Today's Arxiv Papers

(1) [CodePDE: An Inference Framework for LLM-driven PDE Solver Generation](http://arxiv.org/abs/2505.08783v1), 2025-05-13

(2) [HealthBench: Evaluating Large Language Models Towards Improved Human Health](http://arxiv.org/abs/2505.08775v1), 2025-05-13

(3) [SPAT: Sensitivity-based Multihead-attention Pruning on Time Series Forecasting Models](http://arxiv.org/abs/2505.08768v1), 2025-05-13

(4) [Towards Autonomous UAV Visual Object Search in City Space: Benchmark and Agentic Methodology](http://arxiv.org/abs/2505.08765v1), 2025-05-13

(5) [AC-Reason: Towards Theory-Guided Actual Causality Reasoning with Large Language Models](http://arxiv.org/abs/2505.08750v1), 2025-05-13

(6) [DeepMath-Creative: A Benchmark for Evaluating Mathematical Creativity of Large Language Models](http://arxiv.org/abs/2505.08744v1), 2025-05-13

(7) [Probability Consistency in Large Language Models: Theoretical Foundations Meet Empirical Discrepancies](http://arxiv.org/abs/2505.08739v1), 2025-05-13

(8) [NurValues: Real-World Nursing Values Evaluation for Large Language Models in Clinical Context](http://arxiv.org/abs/2505.08734v1), 2025-05-13

(9) [Securing RAG: A Risk Assessment and Mitigation Framework](http://arxiv.org/abs/2505.08728v1), 2025-05-13

(10) [Memorization-Compression Cycles Improve Generalization](http://arxiv.org/abs/2505.08727v1), 2025-05-13

(11) [PWC-MoE: Privacy-Aware Wireless Collaborative Mixture of Experts](http://arxiv.org/abs/2505.08719v1), 2025-05-13

(12) [LLM-based Prompt Ensemble for Reliable Medical Entity Recognition from EHRs](http://arxiv.org/abs/2505.08704v1), 2025-05-13

(13) [Granite-speech: open-source speech-aware LLMs with strong English ASR capabilities](http://arxiv.org/abs/2505.08699v1), 2025-05-13

(14) [VizCV: AI-assisted visualization of researchers' publications tracks](http://arxiv.org/abs/2505.08691v1), 2025-05-13

(15) [Adaptive Schema-aware Event Extraction with Retrieval-Augmented Generation](http://arxiv.org/abs/2505.08690v1), 2025-05-13

(16) [A Social Robot with Inner Speech for Dietary Guidance](http://arxiv.org/abs/2505.08664v1), 2025-05-13

(17) [Revealing economic facts: LLMs know more than they say](http://arxiv.org/abs/2505.08662v1), 2025-05-13

(18) [Enhancing Software Development with Context-Aware Conversational Agents: A User Study on Developer Interactions with Chatbots](http://arxiv.org/abs/2505.08648v1), 2025-05-13

(19) [WixQA: A Multi-Dataset Benchmark for Enterprise Retrieval-Augmented Generation](http://arxiv.org/abs/2505.08643v1), 2025-05-13

(20) [TRAIL: Trace Reasoning and Agentic Issue Localization](http://arxiv.org/abs/2505.08638v1), 2025-05-13

(21) [Visually Guided Decoding: Gradient-Free Hard Prompt Inversion with Language Models](http://arxiv.org/abs/2505.08622v1), 2025-05-13

(22) [Resource-Efficient Language Models: Quantization for Fast and Accessible Inference](http://arxiv.org/abs/2505.08620v1), 2025-05-13

(23) [Automatic Task Detection and Heterogeneous LLM Speculative Decoding](http://arxiv.org/abs/2505.08600v1), 2025-05-13

(24) [Enhancing Thyroid Cytology Diagnosis with RAG-Optimized LLMs and Pa-thology Foundation Models](http://arxiv.org/abs/2505.08590v1), 2025-05-13

(25) [Small but Significant: On the Promise of Small Language Models for Accessible AIED](http://arxiv.org/abs/2505.08588v1), 2025-05-13

(26) [Guiding LLM-based Smart Contract Generation with Finite State Machine](http://arxiv.org/abs/2505.08542v1), 2025-05-13

(27) [The Truth Becomes Clearer Through Debate! Multi-Agent Systems with Large Language Models Unmask Fake News](http://arxiv.org/abs/2505.08532v1), 2025-05-13

(28) [TrialMatchAI: An End-to-End AI-powered Clinical Trial Recommendation System to Streamline Patient-to-Trial Matching](http://arxiv.org/abs/2505.08508v1), 2025-05-13

(29) [InfoPO: On Mutual Information Maximization for Large Language Model Alignment](http://arxiv.org/abs/2505.08507v1), 2025-05-13

(30) [LCES: Zero-shot Automated Essay Scoring via Pairwise Comparisons Using Large Language Models](http://arxiv.org/abs/2505.08498v1), 2025-05-13

(31) [BizChat: Scaffolding AI-Powered Business Planning for Small Business Owners Across Digital Skill Levels](http://dx.doi.org/10.1145/3707640.3731928), 2025-05-13

(32) [Achieving Scalable Robot Autonomy via neurosymbolic planning using lightweight local LLM](http://arxiv.org/abs/2505.08492v1), 2025-05-13

(33) [Large Language Models Meet Stance Detection: A Survey of Tasks, Methods, Applications, Challenges and Future Directions](http://arxiv.org/abs/2505.08464v1), 2025-05-13

(34) [RepCali: High Efficient Fine-tuning Via Representation Calibration in Latent Space for Pre-trained Language Models](http://arxiv.org/abs/2505.08463v1), 2025-05-13

(35) [Strategy-Augmented Planning for Large Language Models via Opponent Exploitation](http://arxiv.org/abs/2505.08459v1), 2025-05-13

(36) [IterKey: Iterative Keyword Generation with LLMs for Enhanced Retrieval Augmented Generation](http://arxiv.org/abs/2505.08450v1), 2025-05-13

(37) [Scalable UAV Multi-Hop Networking via Multi-Agent Reinforcement Learning with Large Language Models](http://arxiv.org/abs/2505.08448v1), 2025-05-13

(38) [Optimizing Retrieval-Augmented Generation: Analysis of Hyperparameter Impact on Performance and Efficiency](http://arxiv.org/abs/2505.08445v1), 2025-05-13

(39) [A document processing pipeline for the construction of a dataset for topic modeling based on the judgments of the Italian Supreme Court](http://arxiv.org/abs/2505.08439v1), 2025-05-13

(40) [An integrated language-vision foundation model for conversational diagnostics and triaging in primary eye care](http://arxiv.org/abs/2505.08414v1), 2025-05-13

(41) [TUMS: Enhancing Tool-use Abilities of LLMs with Multi-structure Handlers](http://arxiv.org/abs/2505.08402v1), 2025-05-13

(42) [Accelerating Chain-of-Thought Reasoning: When Goal-Gradient Importance Meets Dynamic Skipping](http://arxiv.org/abs/2505.08392v1), 2025-05-13

(43) [Towards Contamination Resistant Benchmarks](http://arxiv.org/abs/2505.08389v1), 2025-05-13

(44) [Learning Like Humans: Advancing LLM Reasoning Capabilities via Adaptive Difficulty Curriculum Learning and Expert-Guided Self-Reformulation](http://arxiv.org/abs/2505.08364v1), 2025-05-13

(45) [Alignment Drift in CEFR-prompted LLMs for Interactive Spanish Tutoring](http://arxiv.org/abs/2505.08351v1), 2025-05-13

(46) [Benchmarking AI scientists in omics data-driven biological research](http://arxiv.org/abs/2505.08341v1), 2025-05-13

(47) [Evaluating the Effectiveness of Black-Box Prompt Optimization as the Scale of LLMs Continues to Grow](http://arxiv.org/abs/2505.08303v1), 2025-05-13

(48) [A Practical Introduction to Deep Reinforcement Learning](http://arxiv.org/abs/2505.08295v1), 2025-05-13

(49) [LLM Enhancers for GNNs: An Analysis from the Perspective of Causal Mechanism Identification](http://arxiv.org/abs/2505.08265v1), 2025-05-13

(50) [LLM-Based Detection of Tangled Code Changes for Higher-Quality Method-Level Bug Datasets](http://arxiv.org/abs/2505.08263v1), 2025-05-13

(51) [Enhancing Cache-Augmented Generation (CAG) with Adaptive Contextual Compression for Scalable Knowledge Integration](http://arxiv.org/abs/2505.08261v1), 2025-05-13

(52) [Evaluating LLM Metrics Through Real-World Capabilities](http://arxiv.org/abs/2505.08253v1), 2025-05-13

(53) [Large Language Model Psychometrics: A Systematic Review of Evaluation, Validation, and Enhancement](http://arxiv.org/abs/2505.08245v1), 2025-05-13

(54) [Removing Watermarks with Partial Regeneration using Semantic Information](http://arxiv.org/abs/2505.08234v1), 2025-05-13

(55) [Not that Groove: Zero-Shot Symbolic Music Editing](http://arxiv.org/abs/2505.08203v1), 2025-05-13

(56) [A Head to Predict and a Head to Question: Pre-trained Uncertainty Quantification Heads for Hallucination Detection in LLM Outputs](http://arxiv.org/abs/2505.08200v1), 2025-05-13

(57) [Aitomia: Your Intelligent Assistant for AI-Driven Atomistic and Quantum Chemical Simulations](http://arxiv.org/abs/2505.08195v1), 2025-05-13

(58) [CLTP: Contrastive Language-Tactile Pre-training for 3D Contact Geometry Understanding](http://arxiv.org/abs/2505.08194v1), 2025-05-13

(59) [DSADF: Thinking Fast and Slow for Decision Making](http://arxiv.org/abs/2505.08189v1), 2025-05-13

(60) [Fusing Bidirectional Chains of Thought and Reward Mechanisms A Method for Enhancing Question-Answering Capabilities of Large Language Models for Chinese Intangible Cultural Heritage](http://arxiv.org/abs/2505.08167v1), 2025-05-13

(61) [Decoding Neighborhood Environments with Large Language Models](http://arxiv.org/abs/2505.08163v1), 2025-05-13

(62) [Communication Styles and Reader Preferences of LLM and Human Experts in Explaining Health Information](http://arxiv.org/abs/2505.08143v1), 2025-05-13

(63) [Lost in Transmission: When and Why LLMs Fail to Reason Globally](http://arxiv.org/abs/2505.08140v1), 2025-05-13

(64) [Large Language Models for Computer-Aided Design: A Survey](http://arxiv.org/abs/2505.08137v1), 2025-05-13

(65) [Leveraging AI for Productive and Trustworthy HPC Software: Challenges and Research Directions](http://arxiv.org/abs/2505.08135v1), 2025-05-13

(66) [ALOHA: Empowering Multilingual Agent for University Orientation with Hierarchical Retrieval](http://arxiv.org/abs/2505.08130v1), 2025-05-13

(67) ["You Cannot Sound Like GPT": Signs of language discrimination and resistance in computer science publishing](http://dx.doi.org/10.1145/3715275.3732202), 2025-05-12

(68) [Will Your Next Pair Programming Partner Be Human? An Empirical Evaluation of Generative AI as a Collaborative Teammate in a Semester-Long Classroom Setting](http://dx.doi.org/10.1145/3698205.3729544), 2025-05-12

(69) [Are LLMs complicated ethical dilemma analyzers?](http://arxiv.org/abs/2505.08106v1), 2025-05-12

(70) [Visually Interpretable Subtask Reasoning for Visual Question Answering](http://arxiv.org/abs/2505.08084v1), 2025-05-12

(71) [LLMs to Support K-12 Teachers in Culturally Relevant Pedagogy: An AI Literacy Example](http://arxiv.org/abs/2505.08083v1), 2025-05-12

(72) [Beyond Input Activations: Identifying Influential Latents by Gradient Sparse Autoencoders](http://arxiv.org/abs/2505.08080v1), 2025-05-12

(73) [Who's the Leader? Analyzing Novice Workflows in LLM-Assisted Debugging of Machine Learning Code](http://arxiv.org/abs/2505.08063v1), 2025-05-12

(74) [HYPERNYM MERCURY: Token Optimization through Semantic Field Constriction and Reconstruction from Hypernyms. A New Text Compression Method](http://arxiv.org/abs/2505.08058v1), 2025-05-12

(75) [FalseReject: A Resource for Improving Contextual Safety and Mitigating Over-Refusals in LLMs via Structured Reasoning](http://arxiv.org/abs/2505.08054v1), 2025-05-12

(76) [Large Language Models and Arabic Content: A Review](http://arxiv.org/abs/2505.08004v1), 2025-05-12

(77) [Spec2Assertion: Automatic Pre-RTL Assertion Generation using Large Language Models with Progressive Regularization](http://arxiv.org/abs/2505.07995v1), 2025-05-12

(78) [MilChat: Introducing Chain of Thought Reasoning and GRPO to a Multimodal Small Language Model for Remote Sensing](http://arxiv.org/abs/2505.07984v1), 2025-05-12

(79) [Assessing and Mitigating Medical Knowledge Drift and Conflicts in Large Language Models](http://arxiv.org/abs/2505.07968v1), 2025-05-12

(80) [Symbolic Regression with Multimodal Large Language Models and Kolmogorov Arnold Networks](http://arxiv.org/abs/2505.07956v1), 2025-05-12

(81) [Learning Dynamics in Continual Pre-Training for Large Language Models](http://arxiv.org/abs/2505.07796v1), 2025-05-12

(82) [Overflow Prevention Enhances Long-Context Recurrent LLMs](http://arxiv.org/abs/2505.07793v1), 2025-05-12

(83) [Domain Regeneration: How well do LLMs match syntactic properties of text domains?](http://arxiv.org/abs/2505.07784v1), 2025-05-12

(84) [Relative Overfitting and Accept-Reject Framework](http://arxiv.org/abs/2505.07783v1), 2025-05-12

(85) [MLE-Dojo: Interactive Environments for Empowering LLM Agents in Machine Learning Engineering](http://arxiv.org/abs/2505.07782v1), 2025-05-12

(86) [Agent RL Scaling Law: Agent RL with Spontaneous Code Execution for Mathematical Problem Solving](http://arxiv.org/abs/2505.07773v1), 2025-05-12

(87) [Enhancing Code Generation via Bidirectional Comment-Level Mutual Grounding](http://arxiv.org/abs/2505.07768v1), 2025-05-12

(88) [Assessing the Chemical Intelligence of Large Language Models](http://arxiv.org/abs/2505.07735v1), 2025-05-12

(89) [Spoken Language Understanding on Unseen Tasks With In-Context Learning](http://arxiv.org/abs/2505.07731v1), 2025-05-12

(90) [Circuit Partitioning Using Large Language Models for Quantum Compilation and Simulations](http://arxiv.org/abs/2505.07711v1), 2025-05-12

(91) [Codifying Character Logic in Role-Playing](http://arxiv.org/abs/2505.07705v2), 2025-05-12

(92) [PatchTrack: A Comprehensive Analysis of ChatGPT's Influence on Pull Request Outcomes](http://arxiv.org/abs/2505.07700v1), 2025-05-12

(93) [Re$^2$: A Consistency-ensured Dataset for Full-stage Peer Review and Multi-turn Rebuttal Discussions](http://arxiv.org/abs/2505.07920v1), 2025-05-12

(94) [S-GRPO: Early Exit via Reinforcement Learning in Reasoning Models](http://arxiv.org/abs/2505.07686v1), 2025-05-12

(95) [SpecRouter: Adaptive Routing for Multi-Level Speculative Decoding in Large Language Models](http://arxiv.org/abs/2505.07680v1), 2025-05-12

(96) [OnPrem.LLM: A Privacy-Conscious Document Intelligence Toolkit](http://arxiv.org/abs/2505.07672v2), 2025-05-12

(97) [Benchmarking Retrieval-Augmented Generation for Chemistry](http://arxiv.org/abs/2505.07671v1), 2025-05-12

(98) [A Case Study Investigating the Role of Generative AI in Quality Evaluations of Epics in Agile Software Development](http://arxiv.org/abs/2505.07664v1), 2025-05-12

(99) [JobHop: A Large-Scale Dataset of Career Trajectories](http://arxiv.org/abs/2505.07653v1), 2025-05-12

(100) [Neural Brain: A Neuroscience-inspired Framework for Embodied Agents](http://arxiv.org/abs/2505.07634v1), 2025-05-12
